service/src/SysfsDriver.hpp

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright (c) 2015 - 2022, Intel Corporation
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+#ifndef SYSFSDRIVER_HPP_INCLUDE
+#define SYSFSDRIVER_HPP_INCLUDE
+
+#include <string>
+#include <memory>
+#include <vector>
+#include <map>
+#include <functional>
+
+
+namespace geopm
+{
+    /// @brief Class used to implement the SysfsIOGroup base class
+    ///
+    /// This virtual interface can be adapted for each Linux device
+    /// driver.  A concrete implementation can be used to construct a
+    /// SysfsIOGroup object.
+    class SysfsDriver
+    {
+        public:
+            /// @brief Arbitrary buffer size
+            ///
+            /// We're generally looking at integer values much shorter
+            /// than 100 digits in length. The IOGroup performs string
+            /// truncation checks in case that ever changes.
+            static constexpr size_t M_IO_BUFFER_SIZE = 128;
+            /// @brief The properties about a signal or control
+            struct properties_s {
+                std::string name; // The full low level PlatformIO name
+                bool is_writable; // Is control property?
```

Comment:
Is the comment asking a question we need to bottom out on?

Reply:
That's just the question answered by the value of the bool. I'll rephrase the comment